utils.js

```diff
@@ -21,6 +21,21 @@ let chakram = require('chakram');
  */
 let utils = {
     /**
+     * Ensure that a Chakram response object is sane before using it.
+     * Throws if the response is unreasonable.
+     *
+     * @param {ChakramResponse} response Object to check.
+     */
+    assertSaneResponse: function assertSaneResponse(response) {
```

Comment:
After discussing offline, @rhoggSugarcrm and I think it would be good to call this inside of `wrapRequest` in utils.js instead of in index.js.